packages/webpack/src/runtime/runtime.js

```diff
@@ -153,10 +181,14 @@ const findResourceId = (moduleId) => {
 }
 
 /**
- * @callback WrappedRequire
+ * @callback WebpackRequire
  * @param {string} specifier
  */
 
+/**
+ * @typedef {WebpackRequire & Record<string, any>} WrappedRequire
```

Comment:
```suggestion
 * @typedef {WebpackRequire & Record<string, unknown>} WrappedRequire
```

Would this work?  or

```suggestion
 * @typedef {WebpackRequire & Record<PropertyKey, unknown>} WrappedRequire
```

Reply:
they are strings. Im overiding them so not sure if unknown works, but I'll commit and see.